src/geometry/CTiglIntersectionCalculation.h

```diff
@@ -99,6 +112,9 @@ class CTiglIntersectionCalculation
     // gives a reference to the computed wire
     TIGL_EXPORT TopoDS_Wire GetWire(int wireID);
 
+    // gives a reference to the computed vertex
+    TIGL_EXPORT TopoDS_Vertex GetVertex(int vertexID);
```

Comment:
I have to do the conversion, because I can only cache a `TopoDS_Vertex` in the intersectionResult, which is a  `TopoDS_Compound.` To calculate the parameter later on, I need a `gp_Pnt`. If you prefer, I can do the conversion within this function and return a gp_Pnt rather than the TopoDS_Vertex?

If yes, I would suggest using a new name. Unfortunately, `getPoint` is already taken...